pkg/rules/engine/engine_test.go

```diff
@@ -71,38 +72,6 @@ func TestConsumeSources(t *testing.T) {
 		expectedEvent     interface{}
 		config            Config
 	}{
-		{
-			name: "happy path - with one matching selector, parsed event enabled",
```

Comment:
We don't use parsed events anymore, and it seems that this case is the same as "happy path - with one matching selector"